packages/generator-single-spa/src/svelte/templates/yarn.lock

```diff
@@ -0,0 +1,583 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn.lock files should not be templated - this one can be removed

Reply:
Ah, good call